R/server.R

```diff
@@ -22,6 +22,11 @@ server <- function(elts, opt, custom_admin_panel) {
     if (opt$enable_admin_panel)
       shiny::outputOptions(output, "admin_panel.ui", suspendWhenHidden = FALSE)
 
+    if(!is.null(opt$on_session_ended_fun))
+      session$onSessionEnded(function() {
```

Comment:
Just to check, are you sure you want onSessionEnded and not onStop? Might be worth clarifying your documentation in this respect. See https://stackoverflow.com/a/47013916/8454486

Reply:
Hm that's a good question. Right now I don't need to consider pooling my connections for DB use, but for scalability later in dev `onStop` might be better. Maybe there should be two arguments, one for each scenario? Right now I think I only need `onSessionEnded` though.

Reply:
Yeah I think it's a good idea to do both! Good idea to use the pooling once you decide you're going to scale things up to > 5 simultaneous participants.

Reply:
Cool, just give me a nudge when you want me to look again.